lib/vsc/utils/cache.py

```diff
@@ -100,6 +99,12 @@ def __init__(self, filename, retain_old=True, raise_unpickable=False):
                             self.shelf = {}
                     except (OSError, IOError):
                         self.log.raiseException("Could not load pickle data from %s" % (self.filename,))
+                else:
+                    try:
+                        self.shelf = jsonpickle.decode(s)
+                    except ValueError, err:
+                        self.log.error("Cannot decode JSON from %s [%s]", self.filename, err)
+                        self.shelf = {}
```

Comment:
you forgot to ` self.log.info("Cache in %s starts with an empty shelf" % (self.filename,))`